src/util.js

```diff
@@ -4,13 +4,13 @@ import XoCollection from "xo-collection";
 import { find, forEach, map, cloneDeep } from "lodash";
 
 import config from "./_config";
-import Xoconnection from "./XoWithTestHelpers";
+import XoConnection from "./xoWithTestHelpers";
```

Comment:
Be careful with the module name.

Reply:
I renamed the file.

Reply:
My bad.

Can you add a `_` prefix to mark it as an internal utility?

Reply:
Sure :+1: